packages/features/ee/teams/components/TeamInviteListItem.tsx

```diff
@@ -66,7 +65,7 @@ export default function TeamInviteListItem(props: Props) {
     <div className="flex">
       <Avatar
         size="mdLg"
-        imageSrc={getPlaceholderAvatar(team?.logo, team?.name as string)}
+        imageSrc={`${WEBAPP_URL}/team/${team.slug}/avatar.png`}
```

Comment:
Relying on avatar.png instead of using team logo which can be a huge string impacting page size